lua/advdupe2/sv_clipboard.lua

```diff
@@ -509,6 +509,9 @@ local function CreateConstraintFromTable(Constraint, EntityList, EntityTable, Pl
 	local Factory = duplicator.ConstraintType[Constraint.Type]
 	if not Factory then return end
 
+	if not Player:CheckLimit( "constraints" ) then return end
+	if not Player:CheckLimit( "ropeconstraints" ) then return end
```

Comment:
```suggestion
	if not Player:CheckLimit( Constraint.Type=="Rope" and "ropeconstraints" or "constraints" ) then return end
```
I don't remember if Rope is capitalized or not. Need to be checked. Can't do both CheckLimit because if ropeconstraints is at the max, more of other constraints won't be allowed.

Reply:
Yeah thats what i thought at first too but that's how it works in the basegame https://github.com/Facepunch/garrysmod/blob/master/garrysmod/lua/includes/modules/duplicator.lua#L732-L733, i figured it'd be fine in this case too
If needed i can do the checks individually, it wouldn't be only rope, things like sliders, muscles, hydraulics etc count as ropes.